synorchestrator/trs/docs/Verification.md

```diff
@@ -0,0 +1,65 @@
+# Verifying a test parameter file
+
+1. Go to https://staging.dockstore.org/static/swagger-ui/index.html#/extendedGA4GH/toolsIdVersionsVersionIdTypeTestsPost
+2. Click "Try it now"
+3. Provide a "type", this should be self-explanatory.  See the description for allowable values.
+4. Provide the TRS ID for the tool/workflow being verified.  
+For example, the [dockstore-tool-md5sum](https://dockstore.org/containers/quay.io/briandoconnor/dockstore-tool-md5sum:master?tab=info) tool has the TRS ID: "quay.io/briandoconnor/dockstore-tool-md5sum" as shown in the "Info" tab with the label: "TRS CWL Path" or "TRS WDL Path"
+5. Provide the version_id of the tool/workflow to verify.  It can be any version listed in the Version tab of the tool/workflow.  
+[dockstore-tool-md5sum](https://dockstore.org/containers/quay.io/briandoconnor/dockstore-tool-md5sum:master?tab=versions) has the following versions currently:
+1.0.4, master, develop, 1.0.3, latest, 1.0.2, 1.0.1, and 1.0.0.
+6. Provide the "relative_path" of the test parameter file being verified.  The path of the test parameter file is relative to the primary descriptor.  This path can be found using the [files endpoint](https://staging.dockstore.org/static/swagger-ui/index.html#/GA4GH/toolsIdVersionsVersionIdTypeFilesGet) or by viewing the files tab of a tool/workflow such as:
+https://dockstore.org/containers/quay.io/briandoconnor/dockstore-tool-md5sum:1.0.4?tab=files and then further selecting the Test Parameter Files tab and view the right-most "File" dropdown. This relative path must be a test parameter file, providing a descriptor will not work.
+7. Provide the "platform".  Some examples are: HCA, Cromwell, Arvados, etc
+8. Select the "verified" status either as "true" or "false".  Use "true" to verify, "false" to "unverify".
+9. Provide "metadata", this is typically the verifier's identity which can be something like "GA4GH/DREAM Challenge"
+10. Lastly, provide your Dockstore token using the lock icon at the top right of the endpoint
+
+Below is a screenshot of someone verifying the "test.json" test parameter file of the "master" version of the "dockstore-tool-md5sum" tool.  
+
+![thing](Swagger.png)
+
+The curl command results in something like:
+
+`curl -X POST "https://staging.dockstore.org/api/api/ga4gh/v2/extended/quay.io%2Fbriandoconnor%2Fdockstore-tool-md5sum/versions/master/CWL/tests/test.json?platform=Dockstore%20CLI&verified=true&metadata=Phase%201%20GA4GH%20Tool%20Execution%20Challenge" -H  "accept: application/json" -H  "Authorization: Bearer a0b1c1d8e9f9g9h8i8j1k9l9m9n1o1p9q7r2s5tuvwxyz3"`
```

Comment:
Is this a real token? please make it obviously an example just in case

Reply:
it's not a real token, it's basically the letters of the alphabet